docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# How to contribute
+
+## Commit messages
+
+Commit code often and use the imperative mood when describing the nature of the change. This project uses [Conventional Commits](https://www.conventionalcommits.org/) using the [Angular standards](https://github.com/angular/angular/blob/22b96b9/CONTRIBUTING.md#-commit-message-guidelines) as part of the release management strategy.
+
+<details>
+  <summary>View all commit types</summary>
+
+  | Type  | SemVer | Purpose |
+  | ----- | ------ | ------- |
+  | fix | patch | Resolving a bug or issue in existing code |
+  | feat | minor | Introduces a new feature into the codebase |
+  | style | patch | Formatting, whitespace, and other non-functional changes |
+  | chore | patch | Changes that do not affect production |
+  | docs | none | Adding or editing documentation |
+  | refactor | patch | Neither fixes a bug nor adds a feature |
+  | test | none | Adds or updates test files |
+  | ci | patch | Changes to continuous integration |
+  | build | patch | Changes to build files |
+  | BREAKING CHANGE | major | Introduces a breaking API change |
+</details>
+
+```bash
+type(scope): Commit message
```

Comment:
Commit message should start with lowercase, I believe.

Reply:
Conventional commits does not have a standard, only that you are consistent. I can commit to lowercase if that's your preference.

Reply:
Ok, I thought this was the format from the examples I saw and would be enforced in pre-commit hooks. Since you already added it to the formatting instructions, let's stick with it.